### PR TITLE
Added  Clickable Link to Footer Image for Partner Website

### LINKED ISSUE #475: Clickable Footer Logo to Redirect Users to Partner Site
### Issue Description:
The footer image in the application should be clickable and redirect users to the partner's website when clicked. Currently, the image is static and does not link anywhere.

### Steps to Reproduce:
1. Log in to the application.
2. Scroll to the footer section.
3. Click the footer image.

### Expected Behavior:
- Clicking the footer image should open the partner's website in a new tab.

### PR DESCRIPTION
**Pull request by Jash2606**
#### **Issue:**
The footer image is static and does not redirect to the partner's website.

#### **Solution:**
Wrapped the footer image in an anchor tag to make it clickable, redirecting users to the partner's site when clicked.

### CODE REVIEW

**Comment by Jash2606** [comment]
Hey @rithviknishad , Could you please review it !

File: app/page.tsx [hunk 1]
```diff
@@ -193,7 +193,7 @@ export default async function Home() {
               </div>
 
               <div className="lg:col-span-4">
-                <div className="sticky top-0 pt-20">
+                <div className="top-0 pt-20">
```
Comment (rithviknishad) [inline app/page.tsx:196]: Why was this changed? Seems to be unrelated to the issue

**Comment by rithviknishad** [comment]
Other than that, LGTM. This can be merged once conflicts are resolved too.

**Comment by Jash2606** [comment]
> Other than that, LGTM. This can be merged once conflicts are resolved too.

Hey  @rithviknishad , I had Re-solved the conflict , Could you review it and merge it !!

**Comment by rithviknishad** [comment]
@Jash2606 The conflicts were not resolved. 

image

**Comment by Jash2606** [comment]
> @Jash2606 The conflicts were not resolved.
> 
> image

Hey @rithviknishad,

I'm new to open source and struggling to resolve a conflict in my previous PR. I'll raise a new PR within 20 minutes to address the issue. Apologies for any inconvenience caused and thank you for your understanding.
